.github/workflows/upgrade-test.yml

```diff
@@ -88,20 +88,14 @@ jobs:
         env:
           DSPO_NS: data-science-pipelines-operator
 
-      - name: Prepare for Upgrade Testing
-        run: |
-          # Update the KfDef manifest with the latest released version
-          sed -i "s/main/${{ inputs.released-version }}/" ${{ env.RESOURCES_DIR }}/kfdef/kfdef.yaml
-        working-directory: ${{ env.RESOURCES_DIR }}/kfdef
-
-      - name: Print KfDef Manifest Contents
-        run: cat ${{ env.RESOURCES_DIR }}/kfdef/kfdef.yaml
+      - name: Print DataScienceCluster Manifest Contents
+        run: cat ${{ env.RESOURCES_DIR }}/datasciencecluster/datasciencecluster.yaml
```

Comment:
The workflow replaces the branch in the kfdef uri to the released version here. You might need to add this step back as this is where the operator upgrade happens, but instead just replace `kfdef` with the `datasciencecluster` CR like you've done everywhere else.